pyvista/trame/ui/base_viewer.py

```diff
@@ -8,7 +8,6 @@
 import io
 
 from trame.app import get_server
-from trame.widgets import html
```

Comment:
I am not familiar with trame viewer so I don't know if this import makes sense.

Reply:
cc @annehaley or @alesgenova

Reply:
This is fine to remove, this import is unused.